roles/lib_utils/filter_plugins/oo_filters.py

```diff
@@ -660,6 +660,42 @@ def map_from_pairs(source, delim="="):
     return dict(item.split(delim) for item in source.split(","))
 
 
+def lib_utils_oo_get_node_labels(source, hostvars=None):
+    ''' Return a list of labels assigned to schedulable nodes '''
+    labels = list()
+
+    # Filter out the unschedulable nodes
+    for host in source:
+        if host not in hostvars:
+            return
+        node_vars = hostvars[host]
+
+        # All nodes are considered schedulable,
+        # unless explicitly marked so
+        if node_vars.get('openshift_schedulable') == False:
```

Comment:
I believe this will resolve to False for hosts where the variable is undefined.  We should allow the node labels to be appended if this variable is undefined if we're defaulting behavior to have each host schedulable.

Reply:
Tested it without setting `openshift_schedulable` anywhere - and it returned `None`, previous Fedora Atomic tests failed on webconsole task due to that.

This change would assume the node schedulable and only exclude it only when this var is set to False

Reply:
You're right, this logic works for none.

The only issue we're going to run into now is string types which don't directly map to booleans, such as 'true'.

Reply:
Right, I'll try out a couple of values, but iirc ansible would convert it to a boolean

Reply:
Ansible would if you weren't reading directly from host vars via plugin.  When a variable is used in a normal way, ansible 'templates' the variable and will smartly decide what is supposed to be a bool or not.